app/views/users/show.html.erb

```diff
@@ -1,5 +1,9 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
+  <%= link_to friendship?(@user)[0], friendship?(@user)[1], method: friendship?(@user)[2], class: 'button'%>
+  <% if !inverse_friendship_with(@user).nil? && inverse_friendship_with(@user).status == false %>
+      <%= link_to 'Reject', friendship_path(inverse_friendship_with(@user)), method: :delete, class: 'button mx-2'%>
+    <% end %>
```

Comment:
- [ ] There should be absolutely NO logic in views. >Views are only for displaying information that became available to it from its corresponding controller not generating it.